src/thunks/fetchPalettes.js

```diff
@@ -0,0 +1,16 @@
+import { setError } from '../actions';
+import { fetchData } from '../utils/api';
+
+export const fetchPalettes = (allProjects) => {
```

Comment:
We should add a `setLoading(true)` and `setLoading(false)` in here as well.

Reply:
Our fetchPalettes thunk is nested within our fetchProjects thunk. The fetchProjects sets setLoading to true, awaits all our palettes from each project and then is setting setLoading to false once everything is complete. I think since it's one process we can have just the one call at the start and end.